docs/customization/api/modifying_endpoints.rst

```diff
@@ -0,0 +1,31 @@
+How to rename an endpoint's path?
+---------------------------------
+
+If you want to change an endpoint's path, you just need to change the ``path`` attribute in your config:
```

Comment:
We should probably mention, that in order to do that, you have to duplicate whole config in the first place

Reply:
thats not part of the task tho

Reply:
not entirely true, as this issue does not existed in the previous version, as everything was in one doc